__tests__/arithmetic.test.js

```diff
@@ -94,51 +93,43 @@ describe('Arithmetic', () => {
       const result = runArithmetic('1+1 - - 5');
       expect(result).toEqual(1 + 1 - -5);
     });
+
+    it('should handle negating a function call', () => {
+      const result = runArithmetic('1 - -max(1, 2)');
+      expect(result).toEqual(1 - -Math.max(1, 2));
+    });
   });
 
   describe('max function', () => {
+    const { max } = Math;
```

Comment:
Is this good practice?

Reply:
I've never seen use like this, but I wouldn't think it's wrong... It probably makes sense in this context